docs/self-hosted-appcircle/configure-server/_redis-domain-caution.mdx

```diff
@@ -0,0 +1,9 @@
+:::caution
+##### New Redis Subdomain Access 
+
+Upgrading from older versions to v3.18.0 or later, requires `redis` subdomain that should be registered before upgrading. For more details, you can checkout the [DNS Configuration](/docs/self-hosted-appcircle/install-server/docker.md#4-dns-settings) document section.
```

Comment:
Change link format:
- Remove the `docs` part
- Remove the `.md`

It should be something like that gotten from the web browser address bar. You can check other relative links in the document for the recommended style.

Reply:
Done ✅